development/contracts/ForkingManager.sol

```diff
@@ -0,0 +1,190 @@
+        bytes memory _disputeCall
+    ) external {
+        require(children[0] == address(0), "Children already created");
+        require(children[1] == address(0), "Children already created");
+        // Charge the forking fee
+        IERC20(forkonomicToken).safeTransferFrom(
+            msg.sender,
+            address(this),
+            arbitrationFee
+        );
+
+        // Create the children of each contract
+        (address forkmanager1, address forkmanager2) = createChildren(
+            address(this)
+        );
+        (address token1, address token2) = IForkonomicToken(forkonomicToken)
+            .createChildren();
+        (address bridge1, address bridge2) = IForkableBridge(bridge)
+            .createChildren();
+        (address zkevm1, address zkevm2) = IForkableZkEVM(zkEVM)
+            .createChildren();
+
+        // retrieve some information from the zkEVM contract
+        bytes32 genesisRoot = IPolygonZkEVM(zkEVM).batchNumToStateRoot(
+            IPolygonZkEVM(zkEVM).lastVerifiedBatch()
+        );
+        string memory _trustedSequencerURL = IPolygonZkEVM(zkEVM)
+            .trustedSequencerURL();
+        string memory _networkName = IPolygonZkEVM(zkEVM).networkName();
+        string memory _version = "0.1.0"; // Todo: get version from zkEVM, currently only emitted as event
+        IPolygonZkEVMGlobalExitRoot _globalExitRootManager = IPolygonZkEVM(
+            zkEVM
+        ).globalExitRootManager();
+        IERC20Upgradeable _matic = IERC20Upgradeable(token1);
+        IVerifierRollup _rollupVerifier = IForkableZkEVM(zkEVM)
+            .rollupVerifier();
+        IPolygonZkEVMBridge _bridgeAddress = IPolygonZkEVMBridge(bridge1);
+        IPolygonZkEVM.InitializePackedParameters
+            memory initializePackedParameters = IPolygonZkEVM
+                .InitializePackedParameters({
+                    admin: IPolygonZkEVM(zkEVM).admin(),
+                    trustedSequencer: IPolygonZkEVM(zkEVM).trustedSequencer(),
+                    pendingStateTimeout: IPolygonZkEVM(zkEVM)
+                        .pendingStateTimeout(),
+                    trustedAggregator: IPolygonZkEVM(zkEVM).trustedAggregator(),
+                    trustedAggregatorTimeout: IPolygonZkEVM(zkEVM)
+                        .trustedAggregatorTimeout()
+                });
+
+        // Initialize the tokens
+        IForkonomicToken(token1).initialize(forkmanager1, forkonomicToken);
+        IForkonomicToken(token2).initialize(forkmanager2, forkonomicToken);
+
+        //Initialize the zkevm contracts
+        uint64 _chainID = (IPolygonZkEVM(zkEVM).chainID() / 2) * 2 + 3;
+        uint64 _forkID = (IPolygonZkEVM(zkEVM).forkID() / 2) * 2 + 3;
+        IForkableZkEVM(zkevm1).initialize(
+            forkmanager1,
+            zkEVM,
+            initializePackedParameters,
+            genesisRoot,
+            _trustedSequencerURL,
+            _networkName,
+            _version,
+            _globalExitRootManager,
```

Comment:
This exit manager is an important contract needs to be cloned as well. this is another todo...